src/components/Explore/AttributesSidebar.tsx

```diff
@@ -0,0 +1,591 @@
+  const [dragOverIndex, setDragOverIndex] = useState<number | null>(null);
+
+  const { favoriteAttributes, toggleFavorite, reorderFavorites } = useFavoriteAttributes({ scene: model });
+
+  const filtersVariable = getFiltersVariable(model);
+  const { filters } = filtersVariable.useState();
+
+  const currentFilters = filters.map((filter) => filter.key);
+
+  // Helper functions for handling selection modes
+  const getSelectedAttributes = (): string[] => {
+    if (isMulti) {
+      return Array.isArray(selected) ? selected : [];
+    }
+    return selected && typeof selected === 'string' ? [selected] : [];
+  };
+
+  const isAttributeSelected = (attribute: string): boolean => {
+    const selected = getSelectedAttributes();
+    return selected.includes(attribute);
+  };
+
+  // Transform options into AttributeItem format with scope information
+  const attributeItems: AttributeItem[] = useMemo(() => {
+    return options
+      .filter((option) => option.value && !ignoredAttributes.includes(option.value))
+      .map((option) => {
+        const value = option.value!;
+        let scope: ScopeType = 'Span';
+        let label = option.label || value;
+
+        if (value.startsWith(RESOURCE_ATTR)) {
+          scope = 'Resource';
+          label = label.replace(RESOURCE_ATTR, '');
+        } else if (value.startsWith(SPAN_ATTR)) {
+          scope = 'Span';
+          label = label.replace(SPAN_ATTR, '');
+        }
+
+        return {
+          label,
+          value,
+          scope,
+        };
+      })
+      .sort((a, b) => a.label.localeCompare(b.label));
+  }, [options]);
+
+  // Filter attributes based on search and scope
+  const filteredAttributes = useMemo(() => {
+    if (selectedScope === 'Favorites') {
+      // For favorites scope, show favorites attributes in their custom order
+      const favoritesItems = favoriteAttributes
+        .map((attrValue) => attributeItems.find((item) => item.value === attrValue))
+        .filter(Boolean) as AttributeItem[];
+
+      // Apply search filter
+      return favoritesItems.filter((item) => item.label.toLowerCase().includes(searchValue.toLowerCase()));
+    }
+
+    return attributeItems.filter((item) => {
+      // Filter by search text
+      const matchesSearch = item.label.toLowerCase().includes(searchValue.toLowerCase());
+
+      // Filter by scope
+      const matchesScope = selectedScope === 'All' || item.scope === selectedScope;
+
+      return matchesSearch && matchesScope;
+    });
+  }, [attributeItems, searchValue, selectedScope, favoriteAttributes]);
+
+  // Select the next favorite attribute if the selected attribute is in the filters (single mode only)
+  useEffect(() => {
+    if (!isMulti && selected && typeof selected === 'string' && currentFilters.includes(selected)) {
+      const currentIndex = filteredAttributes.findIndex((item) => item.value === selected);
+      const nextIndex = currentIndex + 1;
+
+      if (nextIndex < filteredAttributes.length) {
```

Comment:
I think this actually allows selecting an attributes that is in filters if it's the last one:

<img width="357" height="145" alt="Image" src="https://github.com/user-attachments/assets/887cef36-0368-49e7-b372-032ab7b30c9d" />